### ISSUE TITLE
Cache Refresh Error

### ISSUE THREAD

**Issue by zamilskhan** [GluuFederation/oxTrust#733]
Package: gluu-server-3.1.1.x86_64 1-1.beta2.centos6
Steps to reproduce: 
1. Install gluu-server-3.1.1
2. run "yum upgrade" inside chroot
2. add LDAP source info
3. Click Update to see the following page. After that whenever the cache refresh tab is clicked, the same page appears. The stack trace is available @ https://pastebin.com/9EeEWbeB
![cache-refresh-error](https://user-images.githubusercontent.com/7703245/30918203-a6823bea-a3bf-11e7-8a3a-e456a141732c.png)

**Comment by mzico** [comment]
@zamilskhan : couldn't reproduce this issue from this side. CR ran okay. 
Here is my oxTrust version [ using a 8GB VM ]: 

```
GLUU.[root@proxy ~]# cat /opt/jetty-9.3/temp/jetty-localhost-8082-identity.war-_identity-any-9147605995641204587.dir/webapp/META-INF/MANIFEST.MF
Manifest-Version: 1.0
Implementation-Title: oxTrust Server
Implementation-Version: 3.1.1-SNAPSHOT
Archiver-Version: Plexus Archiver
Built-By: tomcat
Implementation-Build: c53c988589e3edaa47ea06df843804e639cf17f4
Implementation-Vendor-Id: org.xdi
Created-By: Apache Maven 3.3.9
Build-Jdk: 1.8.0_121

GLUU.[root@proxy ~]#

```

**Comment by zamilskhan** [comment]
Fixed in package: gluu-server-3.1.1.x86_64 1-1.beta3.centos6
oxTrust Implementation-Build: faf1ea73603d158b89023af4b8c4038a8054a611
